README.md

```diff
@@ -57,7 +57,7 @@ To install necessary developer tools and receive weekly update reminders, execut
 ```
 The script will ask for your **sudo password** and then your **BECOME password**. Those are both the same password, which is your Ubuntu WSL password.
 
-> The reminder frequency can be changed by modifying the `cooldown` value in the `/usr/local/bin/devxp-files/cooldown.sh` script **after** the auto-setup is complete. By default it is set to 7 days.
+> The reminder frequency can be changed by modifying the `cooldown` value in the `$HOME/.local/devxp-files/cooldown.sh` script **after** the auto-setup is complete. By default it is set to 7 days.
```

Comment:
Should be in .local/devxp/bin or .local/devxp/scripts

But incidentally, wouldn't it make sense to use a .cooldown data file, like you do for timestamp, et al?

Alternatively, we can create a .devxp config file that people can edit. We're going to need one for the gh cli extension anyway, so possible reuse the same?

Reply:
- Yes we could put it in a bin or scripts folder.
- The reason why i did not make it a dot file is that i wanted it to be easier to find for a potential user.
- This cooldown script gets called from the rc file so it runs on each terminal startup, i doubt we would want that for the gh cli file?